examples/binance-chain/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "binance-chain",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "parcel serve ./index.html"
+  },
+  "dependencies": {
+    "@binance-chain/bsc-connector": "^1.0.0",
```

Comment:
you can update to latest connector, the yarn.lock file size would be shrunk a lot:)

Reply:
Thanks @sohkai @ackratos

Reply:
Done, thanks!